src/cursor/cur_std.c

```diff
@@ -1019,6 +1019,26 @@ __cursor_config_debug(WT_CURSOR *cursor, const char *cfg[])
     return (0);
 }
 
+/*
+ * __check_format_fixed_string --
+ *     Check if the schema format is a fixed string.
```

Comment:
Is this supposed to be `fixed length string` is the reason we're dropping the word length?